### PR TITLE
Increase size of training data

### PR DESCRIPTION
**Pull request by hamelsmu**
I'm increasing the size of the dataset because our sample size was too low.  In my experiments, this results in better model performance.

### CODE REVIEW

**Comment by inc0** [comment]
Are you sure this won't exhaust resources of training machine?

File: src/preprocess.py [hunk 1]
```diff
@@ -43,7 +44,7 @@ def format_time(val):
 output_dir = "/data/"
 
 base_url = 'https://storage.googleapis.com/codenet/issue_labels/'
-df = pd.concat([pd.read_csv(base_url+f'00000000000{i}.csv.gz') for i in range(1)])
+df = pd.concat([pd.read_csv(base_url+f'00000000000{i}.csv.gz') for i in range(3)])
```
Comment (inc0) [inline src/preprocess.py]: Are you sure this change provides an increase in model accuracy?

**Comment by hamelsmu** [comment]
Thats a good question.  I did most of my experiments offline and I am sure it makes it better.  However, lets bring the results into the pr.

/run-full-test

**Comment by inc0** [comment]
I see, validation loss is way lower, this PR really lowered overfitting! This is awesome!

**Comment by hamelsmu** [comment]
This is a test comment to inspect the payload

/full-test-run foo bar

more things here. Putting an image below

pr
